build/ci/community/wait_demo_pr_checks.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -xe
+set +o pipefail
+
+wait_fot_checks_to_start(){
+  community_operators_branch=$1
+  forked_community_operators_repository=$2
+  gh_pr_checks_command="gh pr checks $community_operators_branch --repo $forked_community_operators_repository"
+  while [ `$gh_pr_checks_command | grep -i pending | wc -l` -eq 0 ]; do
+    sleep 1
+  done
+}
+wait_for_checks_to_complete(){
+  community_operators_branch=$1
+  all_tests_passed=false
+  gh_pr_checks_command="gh pr checks $community_operators_branch --repo $forked_community_operators_repository"
+  repo_pr=`gh pr list --repo $forked_community_operators_repository | grep $community_operators_branch`
+  if [[ "$repo_pr" == *"$community_operators_branch"* ]]; then
+    wait_fot_checks_to_start $community_operators_branch $forked_community_operators_repository
+    while [[ ! "`$gh_pr_checks_command | grep -i summary`" =~ "pass" ]] && [[ ! "`$gh_pr_checks_command | grep -i summary`" =~ "fail" ]]; do
```

Comment:
`$gh_pr_checks_command | grep -i summary` can be in a var

Reply:
then maybe [wrap it in a function](https://unix.stackexchange.com/a/444952)?

Reply:
done